.github/workflows/design-system.yml

```diff
@@ -51,15 +51,20 @@ jobs:
           kubectl config set-context ${KUBE_CLUSTER} --cluster=${KUBE_CLUSTER} --user=deploy-user --namespace=${KUBE_NAMESPACE}
           kubectl config use-context ${KUBE_CLUSTER}
 
+      - name: "Get service pod name"
+        run: |
+          echo "SERVICE_POD=$(kubectl -n ${KUBE_NAMESPACE} get pod -l name=central-digital-product-team-development-service-pod -o jsonpath='{.items[0].metadata.name}');" >> $GITHUB_ENV
```

Comment:
Hi @EmilyHazlehurst , this looks good. I have just a minor question, is the semi-colon ok here?

If I test this locally: 

```
KUBE_NAMESPACE=justice-gov-uk-design-system
echo "SERVICE_POD=$(kubectl -n ${KUBE_NAMESPACE} get pod -l name=central-digital-product-team-development-service-pod -o jsonpath='{.items[0].metadata.name}');"
```

It returns:

`SERVICE_POD=central-digital-product-team-development-service-pod-74d46kxhfx;`

Will the semi colon work with the way GH deals with env vars?